### ISSUE TITLE
Bring back the classic wood and stone sounds.

### ISSUE THREAD

**Issue by Tim79000** [minetest/minetest_game#3015]
The new sounds are particular ear grading due to the Minetest sound system making sounds a bit too compressed sounding. The original sounds didn't have this issue due to them not being that crisp. Not to mention I just miss the original sounds. :(

**Comment by NathanSalapat** [comment]
You should be able to look through the git history and get those files, and put them in a mod that depends on default to use the old sounds.

**Comment by Tim79000** [comment]
Yeah, but the issue is that their are no such thing as, replacing sounds with texture packs, I get they are texture packs, but Minecraft has that capability, why can't Minetest?

**Comment by rollerozxa** [comment]
Sound packs.

**Comment by appgurueu** [comment]
Are the "new" sounds really a major regression?

Updating sounds / textures passes as maintainance, so a PR could pass, given it provides sufficient rationale / poll / whatever that reverting is indeed the way to go.

**Comment by Desour** [comment]
> Sound packs.

Sound-packs can only provide non-existing sounds, but not replace sounds.

---

Btw. I've opened a similar issue some time ago (#2755), but it was closed by paramat.

**Comment by appgurueu** [comment]
Feel free to open a PR.

**Comment by ixPost** [comment]
How about adding shade packs too or texture-shade packs.

**Comment by appgurueu** [comment]
The engine supports sound packs as far as I know. Still it might make sense to have sensible default sounds, esp. as UX support for sound packs is poor IIRC.
